### PR TITLE
fix: add missing hover and focus styles

### LINKED ISSUE #61: Review and implement all focus and hover styles
Review all focus and hover styles specified in [the design](https://www.figma.com/file/JVjWAROz9jZBbDaemwom2y/ILDH?node-id=0%3A1) and ensure they are implemented and working properly.

### PR DESCRIPTION
**Pull request by BlueSlug**
* [x] This pull request has been linted by running `npm run lint` without errors
* [x] This pull request has been tested by running `npm run start` and reviewing affected routes
* [x] This pull request has been built by running `npm run build` without errors
* [x] This isn't a duplicate of an existing pull request

## Description

This update adds remaining hover and focus styles specified in the design which were not implemented for the `v1.0.0` release.

## Steps to test

1. Go to the Approaches page
2. Hover over or focus on each of the following elements on the page:
   - the "An Inclusive Design Research Centre project" link above the page header
   - the breadcrumb link to "Home"
   - the "expand all" and "collapse all" buttons
   - each subsection accordion control
3. Click on "Good Practices" to expand it
4. Click on "Follow Accessibility Principles"
5. Once the page is loaded, hover over or focus on the links within the page content

**Expected behavior:** Each of the elements specified above should have a hover/focus indicator which does not look broken.

## Additional information

One of the CSS classes added in this update makes use of the `.topic` selector which is not in the `dev` branch at the time of filing. This will be addressed once #85 is merged into `dev`, at which point this PR will be marked ready for review & merger.

The breadcrumb focus/hover indicators were not specified in the design but were added to improve accessibility.

## Related issues

Resolves #61

### CODE REVIEW

**Comment by jobara** [comment]
@BlueSlug could you please address the merge conflict

**Comment by BlueSlug** [comment]
@jobara ready for more review :)

**Comment by jobara** [comment]
The hover/focus styling for the breadcrumb links in the contrast themes applies the same foreground and background colour. 
Screen Shot 2022-05-24 at 9 26 15 AM

**Comment by BlueSlug** [comment]
@cindyli @jobara ready for more review

File: src/assets/styles/abstracts/_mixins.scss [hunk 1]
```diff
@@ -28,3 +28,29 @@
...
+    color: var(--fl-bgColor, $menuExpandedSelectedColour) !important;
+    margin: unset;
+    outline-color: transparent;
+    padding-block: unset;
+}
+
+@mixin menu-link-focus-hover-wide() {
+    background-color: inherit !important;
+    border-bottom: 0.625rem solid $menuExpandedSelectedBGColour;
+    color: var(--fl-linkColor, currentColor) !important;
```
Comment (jobara) [inline src/assets/styles/abstracts/_mixins.scss:53]: I don't suppose it's possible to remove the use !important? Perhaps a comment to explain what they are for would help if they cannot be removed.

**Comment by cindyli** [comment]
On the figma design, the hover state for links has a thicker underline and a yellow background:
Screen Shot 2022-06-01 at 10 29 11 AM

The actual implementation adds the thicker underline but doesn't add the yellow background:
Screen Shot 2022-06-01 at 10 31 41 AM

**Comment by cindyli** [comment]
When an "article content" box is collapsed, hovering on it thickens its border as the hover style. Clicking on the "article content" to open it, the border of the entire opened box stays thickened. Now, hovering on the expand/collapse button of "article box" doesn't have any hover style.

**Comment by carenwatkins** [comment]
> 

@BlueSlug the behaviour should probably be the same as the categories on root pages when they are expanded (I don't believe this behaviour has been suggested on the Figma file either, just want to clarify before thinking this through). What do you think?

**Comment by BlueSlug** [comment]
> @BlueSlug the behaviour should probably be the same as the categories on root pages when they are expanded (I don't believe this behaviour has been suggested on the Figma file either, just want to clarify before thinking this through). What do you think?

@carenwatkins I agree that the two should be as close to the same behaviour and styling as possible, and I think it would be valuable to have something to further differentiate from the unfocused state (like we did for the nav menu items in the widescreen layout). 

Currently the root page categories do not have any further hover or focus indication once expanded, so I suppose the question also applies to those controls. The primary difference between Article Contents and categories is that for the latter, the "bar" does not expand to encompass its contents, rather staying bar-sized and the contents opening up beneath that. Here's the Approaches page with one of the contrast themes applied:

- Collapsed:
image

- Focused:
image

- Expanded, hover and focus look identical:
image

If we think this may take a few more iterations to get the design to a good place, my suggestion would be to file this as a separate issue and proceed with merging this PR. @cindyli @jobara what do you think of that approach?

**Pull request merged**